aquarius/events/constants.py

```diff
@@ -74,10 +74,10 @@ class EventTypes(SimpleEnum):
             "type": EVENT_DISPENSER_CREATED,
             "text": "DispenserCreated(address,address,uint256,uint256,address)",
         },
-        "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef": {
-            "type": EVENT_TRANSFER,
-            "text": "Transfer(address,address,uint256)",
-        },
+        # "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef": {
```

Comment:
please remove commented out lines if not useful anymore, or otherwise leave an explanation as to why we are keeping them.